tools/wptrunner/wptrunner/wpttest.py

```diff
@@ -120,8 +120,9 @@ def __init__(self, metadata_root, product, debug,
             self["wasm"] = False
         if extras is not None:
             self.update(extras)
+        else:
+            self["headless"] = False
```

Comment:
If `extras` is not None but doesn't contain `headless` we now don't set it, whereas before it was False. Is that expected?

Reply:
Oh, no, that's a mistake.